Week4/MyFavoriteMovies/app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="layout_margin">16dp</dimen>
+    <dimen name="layout_weight">1</dimen>
+    <dimen name="layout_margin_small">8dp</dimen>
+    <dimen name="smallest_layout_width">0dp</dimen>
```

Comment:
You don't have to pull out 0dp sizes if you're using constraint layouts, as this is something that's _implied_ as a non-hardcoded value. 

Anything else should be pulled out, but this is fine either way! GJ on the rest of the dimens!